### ISSUE TITLE
Chopper load

### ISSUE THREAD

**Issue by miracleman51** [Mogli12/GearboxAddon#39]
I'm finding that with when driving the choppers picking up a fairly good crop of grass I can barley go above 1/2mph and there's a sort of 'jerky' motion.  I think you may need adjust the power needed to chop grass or maybe have a sort of 'inertia' in the chopper unit to ensure it deals with larger 'clumps' of grass better?

Otherwise fantastic job.

**Comment by miracleman51** [comment]
Im also finding that PTO driven implements like loading wagons do not tend to 'load up' the tractor when picking up grass or have much relation to the speed at which you drive.

**Comment by Mogli12** [comment]
Can you please add some more details? In this case the log.txt would be really useful and a screen shot.

**Comment by miracleman51** [comment]
Screenshot probably wont show much, but I tested it with a variety of choppers/tractors (both standard and mods)..

For instance Im running the map Klettenberg | NKB-Modding | Version 1.1.2 with soilmod.  I mowed a field at the relevant soilmod nutrient levels to get ~90-100% yield the the Kuhn mounted front and back mowers, then raked it with the Kuhn GA 8521 rake.  When I tried to then pick it up with the Krone Big X1100 forage harvester and the easyflow 300 I cant get the chopper to go above ~2mph while harvesting it. (fyi i was not filling a trailer as I was testing it on its own).

I found that when using a variety of the loading wagons, it did not seem to matter if you were actually picking up a swath or not to the load on the tractor.

[log.txt](https://github.com/Mogli12/GearboxAddon/files/369741/log.txt)
